yklua/bench.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+
+set -eu
+
+lua=$1
+
+if [ ! -f "${lua}" ]; then
+    printf "Cannot find %s executable!\n" "${lua}"
+    exit 1
+fi
+
+hyperfine --warmup 10 "${lua} ./yklua/fannkuchredux.lua 3"
```

Comment:
Not sure `warmup` is very useful, at least not 10 times? I'd probably prefer `--runs 10` rather than `--warmup 10`, although I haven't looked at it deeply.

Reply:
If we do `--warmup` I'd suggest `1` is probably enough, at least for most purposes.

Reply:
added `--runs` :point_right:  c61474998ea33d1daaff43bba28f140bdc911fe8

Reply:
done :point_right: 0f0eabf1a321c39a7c65db9d45b8ba14176aa4f8